### PR TITLE
Feature/Input validation (configure withdraw) - DAO-585 

### PR DESCRIPTION
**Pull request by Fabricevladimir**
- validation for configureWithdraw input fields
- update translation strings for New Withdraw form buttons
- fixed validation bug in New Deposit

### CODE REVIEW

**Review by RakeshUP — APPROVED** [review]

File: packages/web-app/src/containers/configureWithdraw/index.tsx [hunk 1]
```diff
@@ -103,17 +112,27 @@ const ConfigureWithdrawForm: React.FC = () => {
...
+              <StyledInput
+                mode={error ? 'critical' : 'default'}
+                name={name}
+                type="number"
+                value={value}
+                onBlur={onBlur}
+                onChange={onChange}
+              />
+              {error?.message && (
+                <AlertInline label={error.message} mode="critical" />
```
Comment (WissenIstNacht) [inline packages/web-app/src/containers/configureWithdraw/index.tsx:133]: So here we're not (yet) checking whether or not `input amount > tokens in treasury`, right?

File: packages/web-app/src/containers/depositForm/index.tsx [hunk 2]
```diff
@@ -136,35 +142,21 @@ const DepositForm: React.FC = () => {
...
     },
-    [errors.tokenAddress, provider, t, tokenAddress, tokenBalance]
+    [errors.tokenAddress, getValues, provider, t]
   );
 
   /*************************************************
    *             Callbacks and Handlers            *
    *************************************************/
   const handleMaxClicked = useCallback(
     (onChange: (event: unknown[]) => void) => {
```
Comment (WissenIstNacht) [inline packages/web-app/src/containers/depositForm/index.tsx]: `React.ChangeEvent<HTMLInputElement>`?

**Review by WissenIstNacht — APPROVED** [review]
Thanks man, seems to work just fine! :)

File: packages/web-app/src/containers/configureWithdraw/index.tsx (hunk 1 above)
Comment (Fabricevladimir) [reply packages/web-app/src/containers/configureWithdraw/index.tsx:133]: Right

File: packages/web-app/src/containers/depositForm/index.tsx (hunk 2 above)
Comment (Fabricevladimir) [reply packages/web-app/src/containers/depositForm/index.tsx]: Ah, of course. Thanks for catching that

**Pull request merged**